stock_picking_group_by_partner_by_carrier_sale_line_position/models/stock_picking.py

```diff
@@ -1,18 +0,0 @@
-# Copyright 2021 Camptocamp SA
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
-
-from odoo import models
-
-
-class StockPicking(models.Model):
-    _inherit = "stock.picking"
-
-    def _get_sorted_moves(self):
-        self.ensure_one()
-        return self.move_lines.sorted(
-            lambda m: m.sale_line_id.order_id.id * 1000 + m.sale_line_id.position
-        )
-
-    def _get_sorted_move_lines(self):
-        self.ensure_one()
-        return self.move_line_ids.sorted(lambda l: l.move_id.sale_line_id.position)
```

Comment:
I do not understand this change (I don't know this module very well too), could you explain why we don't need to sort anymore?

Reply:
Good day @sebalix, I could not find any place that call those methods.

Good day @TDu, Could you please help to advise the purpose of the methods when adding with
https://github.com/OCA/stock-logistics-reporting/pull/155/files#diff-5e837d58a39e022c613d0595b6d430527a060110365d3e2a1e5e76e612553d93R1-R18?

https://github.com/search?q=org%3AOCA+_get_sorted_moves&type=code
https://github.com/search?q=org%3AOCA+_get_sorted_move_lines&type=code
https://github.com/odoo/odoo/search?q=_get_sorted_moves&type=commits
https://github.com/odoo/odoo/search?q=_get_sorted_move_lines&type=commits